Update README.md

Alex061421 opened the pull request:
Changed the readme for the discrepancy.

Alex061421 committed:
Update README.md

Changed the readme for the discrepancy.

lcoombe reviewed (commented):
Thanks Alex! 
Think we're just missing one last `train.py` -> `train_tAMPer.py` on line 133 of the README.md - then we should be good to merge!

Alex061421 committed:
Corrected readme discrepancies

Alex061421 committed:
Update README.md

Added the additional information regarding localcolabfold and colabfold to reiterate the need for --amber (relaxation) and zipping the results for structure predictions.
